lib/pos_blockchain_test.go

```diff
@@ -1428,12 +1444,257 @@ func TestCanCommitGrandparent(t *testing.T) {
 	// TODO: What other cases do we really need tested here?
 }
 
-func TestCommitGrandparent(t *testing.T) {
-	t.Skip("Skipping TestCommitGrandparent")
+func TestRunCommitRuleOnBestChain(t *testing.T) {
```

Comment:
It'll be good add a comment here to describe the what this test verifies, and the list of cases. This way, it's easier for future readers to eyeball the test.